content/scout/policy/_index.md

```diff
@@ -54,10 +54,10 @@ image up-to-dateness.
 
 Docker Scout ships the following out-of-the-box policies:
 
-- [Fixable critical and high vulnerabilities](#fixable-critical-and-high-vulnerabilities)
-- [Copyleft licenses](#copyleft-licenses)
-- [Outdated base images](#outdated-base-images)
-- [High-profile vulnerabilities](#high-profile-vulnerabilities)
+- [No fixable critical and high vulnerabilities](#no-fixable-critical-and-high-vulnerabilities)
```

Comment:
The actual name contains an `or`: `No fixable critical or high vulnerabilities`

Reply:
Thanks - updated everywhere